frontend/src/components/Trigger.tsx

```diff
@@ -20,7 +20,7 @@ import FormControlLabel from '@material-ui/core/FormControlLabel';
 import MenuItem from '@material-ui/core/MenuItem';
 import * as React from 'react';
 import { classes, stylesheet } from 'typestyle';
-import { ApiTrigger } from '../apis/job';
```

Comment:
Is this trigger being used by both V1 and V2 recurring run? Do we want to preserve a separate trigger for V1?

Reply:
v1 and v2 trigger has the same functionality, so we don't have to preserve the trigger for v1.